api/document/tests/json_importer/annotations_test.py

```diff
@@ -0,0 +1,43 @@
+from document.json_importer.annotations import derive_annotations
+from document.json_importer.importer import convert_node
+from document.models import ExternalLink, FootnoteCitation
+
+from .importer_test import PARA_WITH_LINK
```

Comment:
What do you think about duplicating (or simplifying and duplicating) this fixture into this test file? I think ideal unit tests are self-contained; needing to look in a second file to understand these tests is a layer of indirection we can probably avoid.